avionics/sensors.cpp

```diff
@@ -227,6 +227,7 @@ void initSensors(InitStatus *status)
     #endif
     SerialCamera.begin(CameraBaud);
     while (!SerialCamera) {}
+    delay(100);
```

Comment:
To be safe why don't we increase this? It's during initialization so time delays shouldn't matter. I suggest 5 seconds (5000).

Reply:
i agree longer delay is good